.github/workflows/pr-comment.yaml

```diff
@@ -0,0 +1,24 @@
+name: PR comment
+on: issue_comment
+
+jobs:
+  pr_commented:
+    name: PR comment workflow
+    if: |
+      github.event.issue.pull_request &&
+      github.repository == 'adobe/react-spectrum' &&
+      github.event.comment.author_association == 'MEMBER'
```

Comment:
Open to opinions to this restriction. I can make it more restrictive by checking if the comment author is within a specific list of users (e.g. `contains([fromJson('["LFDanLu", "devongovett", etc...]' , github.actor)`) but I figured this might be restrictive enough since MEMBER is users in the adobe org

Reply:
Allowing Adobe org members seems fine to me.

Reply:
Maybe for testing, you restrict it to just your username.

Reply:
kk updated